app/browser/api/ledger.js

```diff
@@ -1113,6 +1113,37 @@ const pageDataChanged = (state, viewData = {}, keepInfo = false) => {
   }
 
   state = addNewLocation(state, location, tabId, keepInfo)
+  appActions.onCheckBrowserActivityTime()
```

Comment:
Do we need appAction? you have state that you need in this function

Reply:
could you have large publisher set with less then 30min?

Reply:
I guess if you set visit time to min, then table could be quite big before you reach 30min. Let's keep it like you have it